src/common/link.js

```diff
@@ -49,6 +49,14 @@ export class BlockLink extends Component {
     const Component = tagName
     return (
       <Component
+        ref={ref => {
+          // FIXME: when https://github.com/facebook/react/issues/8529 is fixed,
+          // remove and use onAuxClickCapture.
+          // In Chrome ^55, middle-clicking triggers auxclick event instead of click
+          if (ref != null) {
```

Comment:
Per the [doc](https://facebook.github.io/react/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element), it can only be a reference or `null`: stricter test.